examples/faas/src/main.rs

```diff
@@ -12,23 +12,31 @@ fn main() {
         .format(|buf, record| writeln!(buf, "{}", record.args()))
         .init();
     let sim = Simulation::new(1);
-    let mut serverless = ServerlessSimulation::new(sim, None, None, None);
+    let mut serverless = ServerlessSimulation::new(sim, Default::default());
```

Comment:
Предлагаю везде для краткости называть `ServerlessSimulation` как `sim`. О слово `serverless` больше спотыкаешься при чтении кода.

```suggestion
    let mut sim = ServerlessSimulation::new(Simulation::new(1), Default::default());
```

Reply:
Переименовал